docs/howtos/releases.md

```diff
@@ -0,0 +1,99 @@
+# Application Services Release Process
+
+## Nightly builds
+
+Nightly builds are automatically generated using a taskcluster cron task.
+
+- The results of the latest successful nightly build is listed here:
+  https://firefox-ci-tc.services.mozilla.com/tasks/index/project.application-services.v2.nightly/latest
+- The latest nightly decision task should be listed here:
+  https://firefox-ci-tc.services.mozilla.com/tasks/index/project.application-services.v2.branch.main.latest.taskgraph/decision-nightly
+- If you don't see a decision task from the day before, then contact releng.  It's likely that the cron decision task is broken.
+
+## Release builds
+
+Release builds are generated from the `release-vXXX` branches.  Whenever a commit is pushed there,
+we build all artifacts needed for the release.  Once we're ready to publish the release, we run the
+taskcluster release promotion action which signs and publishes the artifacts.
+
+TODO: explain release-promotion more.
```

Comment:
Here's [a link](https://firefox-source-docs.mozilla.org/taskcluster/release-promotion.html) to the Gecko docs on the topic if you want to draw inspiration.